### PR TITLE
Firefox Awesome Bar Cheatsheet in JSON

### PR DESCRIPTION
**Pull request by domjacko**
As requested, Cheatsheet is now created in JSON format which will eventually supersede PR #1151.

------
IA Page: https://duck.co/ia/view/awesome_bar_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/firefox_awesome.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "description": "Firefox Awesome Bar",
+    "metadata": {
+        "sourceName": "Mozilla Support",
+        "sourceUrl": "https://support.mozilla.org/kb/awesome-bar-keyboard-shortcuts"
+    },
+    "section_order": ["Power-ups"],
+    "sections": {
+        "Power-ups": [{
+            "val": "Search for matches in your browsing history",
+            "key": "[^]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/firefox_awesome.json]: @domjacko if there's no spaces (" ") in the `key` we actually wrap the whole thing in `<code></code>` so for all these, you can actually remove the `[]`

File: share/goodie/cheat_sheets/tmux.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
             "val": "Bottom of history",
             "key": "Ctrl-B M-<"
         }, {
-            "val": "Top of histroy",
+            "val": "Top of history",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/tmux.json]: :+1:

File: share/goodie/cheat_sheets/firefox_awesome.json (hunk 1 above)
Comment (domjacko) [inline share/goodie/cheat_sheets/firefox_awesome.json]: This is done now :)

**Comment by moollaza** [comment]
@domjacko sorry it looks like this PR was branched from your original PR?

I would suggest you rebase this PR onto the current master branch, or `git cherry-pick` these commits onto a new branch and make a new PR. Whatever is easiest for you :)

File: share/goodie/cheat_sheets/firefox_awesome.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+    "name": "firefox_awesome",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/firefox_awesome.json]: Let's change the name to "Firefox Awesome Bar" and remove the description

File: share/goodie/cheat_sheets/firefox_awesome.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    "description": "Firefox Awesome Bar",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/firefox_awesome.json]: Also, please add in the new `id` param: 

```json
id: "awesome_bar_cheat_sheet"
```

This correllates with the IA page I just made.. https://duck.co/ia/view/awesome_bar_cheat_sheet

**Comment by moollaza** [comment]
One more thing, lets set the filename to "awesome bar.json" and if you'd like, create a symlink to this file  with the name "firefox awesome bar.json" in the repo. That way both of those phrases will trigger this IA.

**Pull request closed without merging**